CHANGELOG.rst

```diff
@@ -1,6 +1,7 @@
-**2.4.0 - 04/10/24**
+**2.4.0 - FIXME ADD DATE**
```

Comment:
friendly comment to make sure to update this

Reply:
Yeah this is going into an epic branc which we'll need to update this when _that_ gets merged